client/my-sites/current-site/index.jsx

```diff
@@ -105,6 +106,39 @@ class CurrentSite extends Component {
 
 	previewSite = ( event ) => this.props.onClick && this.props.onClick( event );
 
+	renderViewLink() {
```

Comment:
Minor minor thing: can we rename this to something even more explicit that convey intent, like `renderSitePreviewLink` or `renderSiteViewLink`?